src/usr/local/emhttp/plugins/file.activity/data.php

```
@@ -45,4 +49,24 @@
     return $response->withHeader('Content-Type', 'application/json');
 });
 
+$app->get("{$prefix}/locales", function (Request $request, Response $response, $args) {
```

PHPStan (2.1.15)

Parameter $request of anonymous function has invalid type Psr\Http\Message\ServerRequestInterface.


(class.notFound)